package exceptionn.in;

public class Demo {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		int i = 0;
		int j = 0;
		
		try {
			j = 18/i;
			
		} catch (Exception e) {
			// TODO: handle exception
			System.out.println("Something went wrong.."+ e);
		}
		
		System.out.println(j);
		System.out.println("The End");

	}

}
